var/spack/repos/builtin/packages/ampl/package.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

Comment:
All the packages start with:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
# Spack Project Developers. See the top-level COPYRIGHT file for details.
#
# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```
Please restore the extra line :slightly_smiling_face:

Reply:
I removed it because it was still giving me the general license error and I was trying to match it with what was in cmd/license.py.  Once I removed it, I've now passed the tests.  I will add the line back in and commit again.